tests/west/project/test_project.py

```diff
@@ -502,13 +508,7 @@ def test_update(west_init_tmpdir):
 
     # Add commits to the upstream special repos
     remotes = west_init_tmpdir.join('..', 'repos')
-    for r in remotes.join('manifest'), remotes.join('west'):
-        add_commit(str(r), 'test-update-upstream')
-
-    # Check that updating the manifest repository gets the upstream commit
-    cmd('update --update-manifest')
-    assert head_subject('.west/manifest') == 'test-update-upstream'
-    assert head_subject('.west/west') == west_prev  # Unaffected
+    add_commit(remotes.join('west'), 'test-update-upstream')
```

Comment:
Why bother to keep adding the commit here if nothing further is being tested?

Reply:
Deleted.